packages/core/cms/faststore/content-types.json

```diff
@@ -100,6 +100,104 @@
+                      "type": "string",
+                      "default": "VTEX Commerce"
+                    },
+                    "id": {
+                      "title": "ID",
+                      "type": "string"
+                    },
+                    "url": {
+                      "title": "Organization URL",
+                      "type": "string",
+                      "default": "https://vtex.com"
+                    },
+                    "sameAs": {
+                      "title": "Organization URLs",
+                      "type": "array",
+                      "description": "The URL of a page on another website with additional information about your organization. For example, a URL to your organization's profile page on a social media or review site",
+                      "items": {
+                        "type": "string"
+                      }
+                    },
+                    "logo": {
+                      "title": "Organization logo URL",
+                      "description": "A logo that is representative of your organization. Image guidelines: https://developers.google.com/search/docs/appearance/structured-data/organization",
+                      "type": "string"
+                    },
+                    "image": {
+                      "title": "Organization image/logo object",
+                      "description": "An image that is representative of your organization. Image guidelines: https://developers.google.com/search/docs/appearance/structured-data/organization",
+                      "required": ["url"],
+                      "type": "object",
+                      "properties": {
+                        "url": {
+                          "title": "Organization image URL",
+                          "description": "The URL of the image. Make sure the url is crawlable and indexable.",
+                          "type": "string"
+                        },
+                        "caption": {
+                          "title": "Organization image caption",
+                          "description": "A short description of the image.",
+                          "type": "string"
+                        },
+                        "id": {
+                          "title": "Organization image ID",
+                          "description": "A unique identifier for the image. E.g.: #logo",
+                          "type": "string"
+                        }
+                      }
+                    },
+                    "email": {
+                      "title": "Organization email",
+                      "description": "The email address to contact your business",
+                      "type": "string"
+                    },
+                    "telephone": {
+                      "title": "Organization phone",
+                      "description": "A business phone number. Be sure to include the country code and area code in the phone number",
```

Comment:
nitpick: there are some descriptions ending with a period (`.`) and some that doesn't, what do you think about adding it?